src/main/java/org/openstreetmap/atlas/geography/geojson/GeoJsonUtils.java

```diff
@@ -0,0 +1,102 @@
+package org.openstreetmap.atlas.geography.geojson;
+
+import org.openstreetmap.atlas.geography.Location;
+import org.openstreetmap.atlas.geography.Rectangle;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+
+/**
+ * These are utility functions that well help you create GeoJSON!
+ *
+ * @author hallahan
+ */
+public final class GeoJsonUtils
+{
+
+    private GeoJsonUtils()
+    {
+        // Utility class.
+    }
+
+    /**
+     * An iterable of locations will turn into a JsonArray of Longitude, Latitude coordinates.
+     *
+     * @param locations
+     *            An iterable of locations
+     * @return A JsonArray of Longitude, Latitude coordinates.
+     */
+    public static JsonArray locationsToCoordinates(final Iterable<Location> locations)
```

Comment:
I'd prefer you have a separate function for Polygons, that does the geojson specific wrapping for the user. You manually wrap the Polygons coordinates in a lot of places. And have to remember to use ```closedLooop()``` all of which can be handled in this class.